api/test_resources/boxes.json

```diff
@@ -0,0 +1,1379 @@
+      },
+      "witness": {
+        "goType": "generated.ApplicationResponse",
+        "app": {
+          "application": {
+            "created-at-round": 1,
+            "deleted": false,
+            "id": 5,
+            "params": {
+              "approval-program": "CCABADEYQQB1NhoAgAZjcmVhdGUSQQAYgSAxG4ECEkAABUg2GgIXNhoBTLlEQgBONhoAgAZkZWxldGUSQQAINhoBvERCADc2GgCAA3NldBJBAAs2GgEiNhoCu0IAIDYaAIAFY2hlY2sSQQARNhoBIjYaAhW6NhoCEkRCAAEAgQE=",
+              "clear-state-program": "CIEB",
+              "creator": "OKUWMFFEKF4B4D7FRQYBVV3C2SNS54ZO4WZ2MJ3576UYKFDHM5P3AFMRWE",
+              "global-state-schema": {
+                "num-byte-slice": 0,
+                "num-uint": 0
+              },
+              "local-state-schema": {
+                "num-byte-slice": 0,
+                "num-uint": 0
+              }
+            }
+          },
+          "current-round": 8
+        }
+      },
+      "witnessError": null
+    },
+    {
+      "name": "Creator account - not an app account - no params",
+      "request": {
+        "path": "/v2/accounts/LMTOYRT2WPSUY6JTCW2URER6YN3GETJ5FHTQBA55EVK66JG2QOB32WPIHY",
+        "params": [],
+        "url": "http://localhost:8999/v2/accounts/LMTOYRT2WPSUY6JTCW2URER6YN3GETJ5FHTQBA55EVK66JG2QOB32WPIHY",
+        "route": ""
+      },
+      "response": {
+        "statusCode": 200,
+        "body": "{\"account\":{\"address\":\"LMTOYRT2WPSUY6JTCW2URER6YN3GETJ5FHTQBA55EVK66JG2QOB32WPIHY\",\"amount\":500000,\"amount-without-pending-rewards\":500000,\"created-at-round\":1,\"deleted\":false,\"pending-rewards\":0,\"reward-base\":0,\"rewards\":0,\"round\":8,\"status\":\"Offline\",\"total-apps-opted-in\":0,\"total-assets-opted-in\":0,\"total-box-bytes\":0,\"total-boxes\":0,\"total-created-apps\":0,\"total-created-assets\":0},\"current-round\":8}\n"
+      },
+      "witness": {
+        "goType": "generated.AccountResponse",
+        "account": {
+          "account": {
+            "address": "LMTOYRT2WPSUY6JTCW2URER6YN3GETJ5FHTQBA55EVK66JG2QOB32WPIHY",
+            "amount": 500000,
+            "amount-without-pending-rewards": 500000,
+            "created-at-round": 1,
+            "deleted": false,
+            "pending-rewards": 0,
+            "reward-base": 0,
+            "rewards": 0,
+            "round": 8,
+            "status": "Offline",
+            "total-apps-opted-in": 0,
+            "total-assets-opted-in": 0,
+            "total-box-bytes": 0,
+            "total-boxes": 0,
```

Comment:
Even though this is not an app account, we're still are getting the _never_-relevant box totals.